deploy/helms/legion-core/templates/liara/liara-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{ if .Values.liara.enabled -}}

```

Comment:
Can we conditionally include console chart to the legion-core chart instead of checking if it is enabled in every template?

Reply:
Just implement console as separated chart, right? We need to modify ansible role in this case.